enterprise/cypher/acceptance-spec-suite/src/test/scala/org/neo4j/internal/cypher/acceptance/NodeIndexSeekByRangeAcceptanceTest.scala

```diff
@@ -1051,7 +1051,7 @@ class NodeIndexSeekByRangeAcceptanceTest extends ExecutionEngineFunSuite with Cy
       planComparisonStrategy = ComparePlansWithAssertion((plan) => {
         //THEN
         plan should useOperators(IndexSeekByRange.name)
-      }, Configs.AllRulePlanners))
+      }, Configs.AllRulePlanners), params = Map("param" -> Array[Int](1, 2, 3)))
```

Comment:
In my solution, a correct parameter list is also required for EXPLAIN queries, not sure if that is the right thing todo.

Reply:
I wonder if this could be a problem? I suspect there are those that can use the browser to explain queries with nice graphical query plan output that will now no longer be able to do so because the browser does not support query parameters.

Reply:
Ah, that can certainly be a problem. `EXPLAIN` queries should probably be excluded from the check.